tests/all/unit/api/validation/test_createdat.py

```diff
@@ -0,0 +1,57 @@
+import unittest
+from app.models import db
+from datetime import datetime, timedelta, timezone
+from app.factories.session import SessionFactory
+from app.factories.user import UserFactory
+from app.factories.event import EventFactoryBasic
+from app.factories.access_code import AccessCodeFactory
+from app.factories.event_invoice import EventInvoiceFactory
+from app.factories.attendee import AttendeeFactory
+from app.factories.discount_code import DiscountCodeFactory
+from app.factories.order import OrderFactory
+from app.factories.role_invite import RoleInviteFactory
+from app.factories.user_token_blacklist import UserTokenBlacklistFactory
+from tests.all.integration.utils import OpenEventTestCase
+
+
+class TestCreatedatValidation(OpenEventTestCase):
+    def test_createdat(self):
+        """ Validate time : Tests if created_at is set to current time in all models
+        """
+        with self.app.test_request_context():
+            model_factories = [
+                UserFactory,
+                EventInvoiceFactory,
+                AccessCodeFactory,
+                EventFactoryBasic,
+                OrderFactory,
+                RoleInviteFactory,
+                DiscountCodeFactory,
+                AttendeeFactory,
+                SessionFactory,
+                UserTokenBlacklistFactory,
+            ]
+            models_with_utcnow = [
+                AttendeeFactory,
+                DiscountCodeFactory,
+                EventInvoiceFactory,
+            ]
+            for model_factory in model_factories:
+                with self.subTest(model_factory=model_factory):
+                    test_model = model_factory()
+                    db.session.commit()
+                    if model_factory in models_with_utcnow:
+                        current_time = datetime.utcnow().astimezone()
+                    else:
+                        current_time = datetime.now(timezone.utc).astimezone()
```

Comment:
There shouldn't be any difference. Please make them consistent

Reply:
> There shouldn't be any difference. Please make them consistent

There's actually a difference, as you can see in the below image.
Printing in this order: 
![image](https://user-images.githubusercontent.com/39561007/73608714-33178800-45ec-11ea-8f0c-2bcf8a19d699.png)

Output:
![image](https://user-images.githubusercontent.com/39561007/73608696-1da25e00-45ec-11ea-8941-83c75eea0008.png)

Reply:
I never said there isn't a difference. I said there shouldn't be a difference

Reply:
The values am getting are weird, one has date with an offset added and specifying the offset[1]. Other one has no offset set also coming with offset added[2].
There is no python function to standardize these two. 
1) 2020-02-02 20:24:32.051071+05:30
2) 2020-02-02 14:54:32.064461+05:30